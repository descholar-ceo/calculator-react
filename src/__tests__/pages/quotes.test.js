import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../../pages/quotes';

describe('Test quotes page with snapshot', () => {
  it('Render quotes page correctly', done => {
    const quotesPage = renderer.create(<Quotes />).toJSON();

    expect(quotesPage).toMatchSnapshot();
    done();
  });
});
