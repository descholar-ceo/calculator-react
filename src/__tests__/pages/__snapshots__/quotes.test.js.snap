// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test quotes page with snapshot Render quotes page correctly 1`] = `
<div>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
  </p>
</div>
`;
